### ISSUE TITLE
Problems with 'resize' and 'insert_row' (AttributeError: 'HTTPError' object has no attribute 'code')

### ISSUE THREAD

**Issue by stemrich** [burnash/gspread#269]
Hello!
I’ve just run into a problem with an (older) python-script using *gspread* (btw: thanks for this great module!!!). I did update the gspread module and I do use the OAuth method for connection. Never the less I do get an error trying to *resize* a worksheet.
Alternatively I try to simply *insert_row* at the end of the sheet, which gives me the same error.
See below for the error-producing code.

```python
MacBook:Project212 xxxx$ pip freeze | grep gspread
gspread==0.2.5
MacBook:Project212 xxxx$ python
Python 2.7.9 |Anaconda 2.1.0 (x86_64)| (default, Dec 15 2014, 10:37:34) 
[GCC 4.2.1 (Apple Inc. build 5577)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
Anaconda is brought to you by Continuum Analytics.
Please check out: http://continuum.io/thanks and https://binstar.org
>>> 
>>> import json
>>> import gspread
>>> from oauth2client.client import SignedJwtAssertionCredentials
>>> 
>>> json_key = json.load(open('google_login.json'))
>>> scope = ['https://spreadsheets.google.com/feeds']
>>> credentials = SignedJwtAssertionCredentials(json_key['client_email'], json_key['private_key'], scope)
>>> sheet_ref = 'reference2somesheet'
>>> 
>>> gc = gspread.authorize(credentials)
>>> spreadsheet = gc.open_by_key(sheet_ref)
>>> ws_active = spreadsheet.worksheet('worksheet')
>>> 
>>> ws_active.row_count
385
>>> 
>>> ws_active.resize(rows=386, cols=9)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/stef/anaconda/lib/python2.7/site-packages/gspread/models.py", line 507, in resize
    self._element = self.client.put_feed(uri, ElementTree.tostring(feed))
  File "/Users/stef/anaconda/lib/python2.7/site-packages/gspread/client.py", line 282, in put_feed
    if ex.code == 403:
AttributeError: 'HTTPError' object has no attribute 'code'
>>> 
>>> ws_active.insert_row(['a', 'b'], index=386)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/stef/anaconda/lib/python2.7/site-packages/gspread/models.py", line 549, in insert_row
    self.add_rows(1)
  File "/Users/stef/anaconda/lib/python2.7/site-packages/gspread/models.py", line 514, in add_rows
    self.resize(rows=self.row_count + rows)
  File "/Users/stef/anaconda/lib/python2.7/site-packages/gspread/models.py", line 507, in resize
    self._element = self.client.put_feed(uri, ElementTree.tostring(feed))
  File "/Users/stef/anaconda/lib/python2.7/site-packages/gspread/client.py", line 282, in put_feed
    if ex.code == 403:
AttributeError: 'HTTPError' object has no attribute 'code'
>>> 

```

Unfortunatelly I have no clue what the reason for this behaviour is. Any help greatly appreciated :)

**Comment by msuozzo** [comment]
I believe this was fixed by 2789ff5328ded843c29a9ecb10aa9921069352e1

**Comment by msuozzo** [comment]
Just pushed a fix (f3cb9f8) for this.
